localstack/services/lambda_/lambda_api.py

```diff
@@ -2338,7 +2338,8 @@ def create_code_signing_config():
     signing_profile_version_arns = data.get("AllowedPublishers").get("SigningProfileVersionArns")
 
     code_signing_id = "csc-%s" % long_uid().replace("-", "")[0:17]
-    arn = arns.code_signing_arn(code_signing_id)
+    # See note above on the use of `aws_stack.get_region()` and `get_aws_account_id()`
+    arn = arns.code_signing_arn(code_signing_id, get_aws_account_id(), aws_stack.get_region())
```

Comment:
For the legacy Lambda provider, the effort involved to refactor the codebase is substantial so we make an exception and use the `get_region()` and `get_aws_account_id()` helpers.